### PR TITLE
Example sg kube

### PR DESCRIPTION
**Pull request by vburckhardt**
### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -140,18 +140,12 @@ resource "ibm_is_security_group_rule" "security_group_rule" {
     # Conditianally add content if sg has icmp
     content {
       type = lookup(
-        lookup(
-          each.value,
-          "icmp"
-        ),
+        each.value["icmp"],
```
Comment (daniel-butler-irl) [inline main.tf:143]: This is just a syntax change right, no other effect on the main module?

Comment (vburckhardt) [reply main.tf:143]: correct, tflint starting to flag those

**Review by daniel-butler-irl — APPROVED** [review]

**Pull request merged**